src/chain_spec.rs

```diff
@@ -5,7 +5,7 @@ use types::{ChainSpec, Config, MainnetEthSpec, MinimalEthSpec};
 
 use crate::{networks::SupportedNetworks, DepositError};
 
-pub fn chain_spec_for_network(network: SupportedNetworks) -> Result<ChainSpec, DepositError> {
+pub fn chain_spec_for_network(network: &SupportedNetworks) -> Result<ChainSpec, DepositError> {
```

Comment:
you should make a note somewhere in the readme that the library portion of this crate isn't following semver, or make a major release

Reply:
Added note in https://github.com/ChorusOne/eth-staking-smith/pull/54/commits/8690bef27260423193ea8fc55259284f33dbfc64